README.rst

```diff
@@ -13,16 +14,22 @@ A `structured concurrent`_, async-native "`actor model`_" built on trio_ and mul
 .. _structured concurrent: https://trio.discourse.group/t/concise-definition-of-structured-concurrency/228
 
 
-``tractor`` is an attempt to bring trionic_ `structured concurrency`_ to
-distributed multi-core Python; it aims to be the Python multi-processing
-framework *you always wanted*.
+``tractor`` is a `structured concurrent`_ "`actor model`_" built on trio_ and multiprocessing_.
+
+It is an attempt to pair trionic_ `structured concurrency`_ with
+distributed Python. You can think of it as a ``trio`` *-across-processes*
+or simply as a replacement for the stdlib's `multiprocessing` but built
+on async primitives for IPC.
+
+``tractor``'s nurseries lets you spawn ``trio`` *"actors"*: new Python
```

Comment:
Super nitpicky, but I think `nurseries let you` (instead of `lets`) is the correct grammar?

Reply:
Pretty sure this is fixed now.